docs/other/install-cvmfs-stratum1.md

```diff
@@ -0,0 +1,338 @@
+```
+
+On EL7-based systems (apache httpd 2.4 and later) replace the "Order allow, deny" and "Allow from all" to "Require all granted".
+
+If you will be serving cern.ch repositories, it has the same problem; replace opensciencegrid.org above with cern.ch. If you need to serve both opensciencegrid.org and cern.ch contact Dave Dykstra to discuss the options.
+
+Then enable apache:
+
+```
+[root@client ~] $ chkconfig httpd on 
+[root@client ~] $ service httpd start
+```
+
+## Configuring frontier-squid
+
+Put the following in `/etc/squid/customize.sh` after the existing comment header:
+
+```
+awk --file `dirname $0`/customhelps.awk --source '{
+
+# cache only api calls 
+insertline("^http_access deny all", "acl CVMFSAPI urlpath_regex ^/cvmfs/\[^/\]\*/api/") insertline("^http_access deny all", "cache deny CVMFSAPI")
+
+# port 80 is also supported, through an iptables redirect 
+setoption("http_port", "8000 accel defaultsite=localhost:8080 no-vhost") setoption("cache_peer", "localhost parent 8080 0 no-query originserver")
+
+# allow incoming http accesses from anywhere \# all requests will be forwarded to the originserver 
+commentout("http_access allow NET_LOCAL") insertline("^http_access deny all", "http_access allow all")
+
+# do not let squid cache DNS entries more than 5 minutes 
+setoption("positive_dns_ttl", "5 minutes")
+
+# set shutdown_lifetime to 0 to avoid giving new connections error \# codes, which get cached upstream 
+setoption("shutdown_lifetime", "0 seconds")
+
+# turn off collapsed_forwarding to prevent slow clients from slowing down faster ones
+setoption("collapsed_forwarding", "off")
+
+print }'
+```
+
+
+On an RHEL7 system, make sure that iptables-services is installed and enabled:
+
+```
+[root@client ~] $ yum -y install iptables-services 
+[root@client ~] $ systemctl enable iptables
+```
+
+Forward port 80 to port 8000 (first command is for external, second command for localhost):
+
+```
+[root@client ~] $ iptables -t nat -A PREROUTING -p tcp -m tcp --dport 80 -j REDIRECT --to-ports 8000 
+[root@client ~] $ iptables -t nat -A OUTPUT -o lo -p tcp -m tcp --dport 80 -j REDIRECT --to-ports 8000 
+[root@client ~] $ service iptables save
+```
+
+On RHEL7 also set up the the same port forwarding for IPv6 (unfortunately it is not supported on RHEL6):
+
+```
+[root@client ~] $ ip6tables -t nat -A PREROUTING -p tcp -m tcp --dport 80 -j REDIRECT --to-ports 8000
+[root@client ~] $ ip6tables -t nat -A OUTPUT -o lo -p tcp -m tcp --dport 80 -j REDIRECT --to-ports 8000
+[root@client ~] $ service ip6tables save
+```
+
+
+Enable frontier-squid:
+
+```
+[root@client ~] $ chkconfig frontier-squid on
+[root@client ~] $ service frontier-squid start
+```
+
+Note: the above configuration is for a single squid thread, which is fine for 1Gbit/s and possibly 2Gbit/s, but if higher bandwidth is needed, see the [instructions for running multiple squid workers](https://twiki.cern.ch/twiki/bin/view/Frontier/InstallSquid#Running_multiple_squid_workers).
```

Comment:
The squid doc has been converted already, fix the link

Reply:
This is a link to the CERN twiki, we don't cover this in our docs.